content/blog/Announcing-the-winner-of-Kenneth_Gonsalves_Award_2016-Dr.Ajith_Kumar_B.P.md

```diff
@@ -12,17 +12,17 @@ Summary: Announcing the winner of "Kenneth Gonsalves Award 2016"-Dr.Ajith Kumar
 
 ![Dr.Ajith Kumar B.P](https://in.pycon.org/2015/images/ajith.jpg)
 
- Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and PhD in Nuclear Physics from University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
+ Ajith Kumar B.P. has been working as a scientist, at Inter University Accelerator Centre, New Delhi, since 1987. He has done his M.Sc and Ph.d. in Nuclear Physics from the University of Calicut and M.Sc in Accelerator Physics, from University of Manitoba.  His work mainly involved in the design and development of particle accelerators and associated experimental facilities.
 
-For the past two decades, he has been involved in promoting Free Software tools in Education, as well as putting some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus, and has also written a book on Python"Python for Education" for this purpose.
+For the past two decades, he has been involved in promoting Free Software tools in Education. He has put some efforts to make free and open source tools popular among the academic community , which include Python in their syllabus. He has also written a book on Python "Python for Education" for this purpose.
```

Comment:
- It should be `Software Tools`.
- It should be `community,` not `community ,`.
- Change `include Python` to `includes Python`.